src/index.tsx

```diff
@@ -1,25 +1,31 @@
-import { StrictMode } from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './components/App';
-import { ThemeProvider } from './theme';
+import { ThemeContext, ThemeProvider } from './theme';
 import { StateProvider } from './state';
 import { getWebInstrumentations, initializeFaro } from '@grafana/faro-web-sdk';
+import { ThemeContext as GrafanaThemeContext } from '@grafana/ui';
 import { TracingInstrumentation } from '@grafana/faro-web-tracing';
 import { ReactIntegration } from '@grafana/faro-react';
+import React, { useContext } from 'react';
+
+const GrafanaTheme = (props: React.PropsWithChildren) => {
```

Comment:
Temporal workaround until we finish #25 and get rid of the custom `ThemeProvider` in favor of `ThemeContext` from `@grafana/ui`.